content/home/src/howto_guides/submit2nmdc.md

```diff
@@ -110,11 +128,99 @@ A ‘Jump to column’ feature is available for searching for specific metadata
 
+##### Find and Replace
+
+TODO: ADD TEST AND IMAGE
+
 ##### Color Legend
 
 [![](../_static/images/howto_guides/portal_guide/color_legend.png)](../_static/images/howto_guides/portal_guide/color_legend.png)
 
 All fields in the metadata template innately fall into one of three color categories: Grey, Yellow, or Purple. Grey, or no highlighted color, indicates optional fields. Required columns are denoted with yellow. These yellow columns must be correctly completed or the submission will not validate. Recommended fields, highlighted purple, are required where applicable. If any of the purple columns provide information relevant to the study, they should be completed. For example, if samples are from a moisture manipulation study, the column ‘watering regimen’ should be filled in to provide context and information about the samples. 
 
-After selecting the validate button, cells will become color coded to indicate invalid and incomplete metadata. Dark pink cells indicate a required cell has been left empty. Light pink indicates that there is an error in the formatting of the information entered into that cell. The ‘Column Information’ tool, described above, provides expected structure patterns and examples of valid metadata in order to help remedy errors and invalid inputs. 
-)
+After selecting the validate button, cells will become color coded to indicate invalid and incomplete metadata. Dark pink cells indicate a required cell has been left empty. Light pink indicates that there is an error in the formatting of the information entered into that cell. The ‘Column Information’ tool, described above, provides expected structure patterns and examples of valid metadata in order to help remedy errors and invalid inputs.
+
+## Field Notes Functionality
+
+https://microbiomedata.org/field-notes/
+
+The NMDC Field Notes App is for mobile device use and is available for beta testing. See (TODO) to sign up!
+
+This app is designed to streamline real-time registration and collection of standardized metadata and field measurements. 
+
+Each section of the app will automatically launch a quick walk-through. You can press the → button to continue the tour or press the X button to end it. To re-initiate the walk through at any time, visit the Settings page and select ‘App Walk-Through’
+
+Available in the app is general fieldwork guidance and recommendations for collecting proper metadata for your sample type(s) to help you plan your fieldwork and metadata collection. We recommend viewing this information before you go out into the field.
+
+TODO: Add screen shot images
+
+The Field Notes App requires ORCiD authentication to access. 
+
+Submissions via Field Notes are available in the Submission Portal as an automatic backup when connected to wifi. The app is available for offline use, but requires internet connection to **start** a submission. 
```

Comment:
Maybe "Submissions via Field Notes are available in the Submission Portal as an automatic backup when connected to wifi" could be "Data entered via Field Notes are automatically synced and available in the Submission Portal when connected to wifi or cellular network".
To be check by @pkalita-lbl , but I believe this sync is active as soon as there is an internet connection, no matter if wifi or else ?

Reply:
@simroux data, or metadata? i worry about confusing here... and study information which isn't really data or metadata also syncs

Reply:
Yeah, I used Data by default but you're right it's not necessarily a good idea...
Maybe "Information entered is" or "Submissions via Field Notes are" (which was the original wording and worked I think) ?